Fails on iOS 16.x in flutter app

Has anyone successfully used this package on iOS 16.x? Running a simple Flutter App causes the following persistent socket send error, details below.

The error (OS Error: No route to host, errno = 65) seems to point to a Local Network access issue, however the App *does* have Local Network Access enabled in Settings.

flutter: \^[[35m[10:47:47.236] [esp_smartconfig] [DEBUG] Creating UDP socket on port 18266<…>
flutter: \^[[35m[10:47:47.261] [esp_smartconfig] [DEBUG] UDP socket on port 18266 successfully created<…>
flutter: \^[[32m[10:47:47.274] [esp_smartconfig] [INFO] Provisioning started<…>
flutter: \^[[31m[10:47:47.283] [esp_smartconfig] [ERROR] Socket error\^[[0m
\^[[31m[exception] SocketException: Send failed (OS Error: No route to host, errno = 65), address = 0.0.0.0, port = 18266\^[[0m
\^[[31m[stacktrace]\^[[0m
#0      _NativeSocket.send (dart:io-patch/socket_patch.dart:1213:34)
#1      _RawDatagramSocket.send (dart:io-patch/socket_patch.dart:2460:15)
#2      Protocol.send (package:esp_smartconfig/src/protocol.dart:129:20)
#3      Protocol.sendBlock._ (package:esp_smartconfig/src/protocol.dart:63:7)
#4      Bottleneck.flow (package:esp_smartconfig/src/bottleneck.dart:22:7)
#5      Protocol.sendBlock (package:esp_smartconfig/src/protocol.dart:75:18)
#6      Protocol.loop (package:esp_smartconfig/src/protocol.dart:98:5)
#7      Provisioner._startProvisioning.<anonymous closure> (package:esp_smartconfig/src/provisioner.dart:204:38)
#8      _Timer._runTimers (dart:isolate-patch/timer_impl.dart:398:19)
#9      _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:429:5)
#10     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.d<…>
flutter: \^[[31m[10:47:47.285] [esp_smartconfig] [ERROR] Socket error: SocketException: Send failed (OS Error: No route to host, errno = 65), address = 0.0.0.0, port = 18266<…>
flutter: \^[[35m[10:47:47.288] [esp_smartconfig] [DEBUG] Destroying isolate<…>
flutter: \^[[32m[10:47:47.289] [esp_smartconfig] [INFO] Provisioning stopped<…>

--- a/README.md
+++ b/README.md
@@ -18,6 +18,14 @@ All non-web platforms are supported. Web platform is not supported mainly becaus
 
 > If you are going to use this library on Desktop platforms make sure that UDP port `18266` is open in firewall for incoming data.
 
+##### Breaking Change iOS 14.6/16
+The breaking change was implemented with 14.6. However, it seems to be the case that the mentioned problem only occurs on devices with iOS version 16. It results in the fact that no connection to the ESP can be established. 
+
+**How to:**
+- You need an Apple-Developer account
+- After the Dev-Account has been approved, you have to make a request via this [form](https://developer.apple.com/contact/request/networking-multicast) to be allowed to use the Multicast Api.
+- Then you can follow this [guide](https://developer.apple.com/forums/thread/663271?answerId=639455022#639455022) (new process)
+
 ## Demo video
 
 [![Demo YouTube video](https://img.youtube.com/vi/yjxtwQ8Xpuo/mqdefault.jpg)](https://youtu.be/yjxtwQ8Xpuo)

